README.md

```diff
@@ -1,2 +1,36 @@
 # jhive-previz
-Processes input data and outputs it in formats required for use in jhive-viz 
+
+This code processes input JWST catalogs from a given DJA field, and outputs a combined data table and metadata file in formats required for use in the J-HIVE Visualization Tool. For detailed documentation of the input and output data structures, visit the [J-HIVE Docs](https://j-hive.github.io/jhive-docs/previz/index.html).
+
+## Installation
+To install this code, clone the repository, enter it, and install using poetry as follows:
+```
+cd jhive-previz
+poetry install
+```
+
+
+## Quickstart
+
+To process the data with the default configuration files in the package, run the following command:
+```
+poetry run jhive_previz
+```
+
```

Comment:
Indicate what happens and where files should be. Put these in separate blocks so that it's clear from a glance (i.e., not in paragraph form)

Reply:
Done!